staked-relayer/src/main.rs

```diff
@@ -258,13 +257,21 @@ async fn main() -> Result<(), Error> {
         ))
     );
     match result {
-        Ok(res) => {
-            println!("{:?}", res);
-        }
+        Ok(_) => Ok(()),
         Err(err) => {
-            println!("Error: {}", err);
-            std::process::exit(1);
+            error!("{:?}", err);
```

Comment:
This error is printed twice (here, and in main). I would suggest removing this `error!` here, and replacing the `eprintln` in `main` with `error!`. But it's up to you, I approved in case you just want to keep it as is

Reply:
`try_join` returns `JoinError` not `Error`, so this is the "real" error from service panics